src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -45,16 +45,19 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
     static _getUrlAttr (el, attr) {
         var attrValue = el.getAttribute(attr);
 
-        if (attrValue === null)
-            return '';
+        // NOTE: In some browsers, elements without the url attribute return the location url
+        // when accessing this attribute directly. See form.action in Edge 25 as an example.
+        var browserFallsBackToLocationUrl = attrValue === null && el[attr];
```

Comment:
Can we make feature detection here, e.g.:
*src/client/feature-detections.js*
```js
 export.emptyUrlAttrFallbacksToTheLocation = NativeMethods.createElement(document, 'a').href === window.location;
```

here:
```
if (attrValue === '' && featureDetection.emptyUrlAttrFallbacksToTheLocation)
    return destLocation.get();		             
```